at-hoangtran/Week3/Day5/Exercise1-Day5.rb

```diff
@@ -0,0 +1,35 @@
+class Student
+  attr_accessor :first_name, :last_name, :list_of_courses
+  def initialize
+    @list_of_courses = []
+  end
+
+  def first_name
+    puts "#{@first_name}"
+  end
+  def last_name
+    puts "#{@last_name}"
+  end
+  def full_name
+    puts "#{@first_name}" + " #{@last_name}"
+  end
+  def add_course(course_name) 
+    unless @list_of_courses.include?(course_name)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/074932088ed5cad5e69983d076619cff8875320e/at-hoangtran/Week3/Day5/Exercise1-Day5.rb#L17"></span>Style/UnlessElse: Do not use <code>unless</code> with <code>else</code>. Rewrite these with the positive case first. (https://github.com/rubocop-hq/ruby-style-guide#no-else-with-unless)</td>
    </tr>
  </tbody>
</table>